integration/run_test

```diff
@@ -37,6 +37,7 @@ run_test() {
   ./integration/test_ei_event "iotx-mock-grpc" "$EVM_SUBSCRIBED_ADDRESS" "" 1
   ./integration/test_ei_event "cfx-mock-http" "$EVM_SUBSCRIBED_ADDRESS" "" 1
   ./integration/test_ei_event "cfx-mock-ws" "$EVM_SUBSCRIBED_ADDRESS" "" 1
+  ./integration/test_ei_event "eth-qae-mock" "$EVM_SUBSCRIBED_ADDRESS" "" 1 "getFlag" '[{"inputs":[],"name":"getFlag","outputs":[{"internalType":"bool","name":"","type":"bool"}],"stateMutability":"view","type":"function"}]'
```

Comment:
Instead of supplying the ABI, it'd be better if we could just give the function signature (either the 4-byte hex or string), similar to how EthTx handles it on the Chainlink node.

Reply:
The difference between EthTx and this initiator is that EthTx will write the value to the function signature, while this initiator will both query and parse the response. Without the ABI we're not able to parse the correct value that the function returns.